src/pages/vue/your-first-app.md

```diff
@@ -1,10 +1,202 @@
+* <strong>A code editor</strong> for... writing code! We are fans of [Visual Studio Code](https://code.visualstudio.com/).
+* <strong>Command-line interface/terminal (CLI)</strong>:
+ * <strong>Windows</strong> users: for the best Ionic experience, we recommend the built-in command line (cmd) or the Powershell CLI, running in Administrator mode.
+ * <strong>Mac/Linux</strong> users, virtually any terminal will work.
+
+
+## Install Ionic Tooling
+Run the following in the command line terminal to install the Ionic CLI (`ionic`), `native-run`, used to run native binaries on devices and simulators/emulators, and `cordova-res`, used to generate native app icons and splash screens:
+
+> To open a terminal in Visual Studio Code, go to Terminal -> New Terminal.
+
+```shell
+$ npm install -g @ionic/cli@testing native-run cordova-res
+```
+
+> The `-g` option means _install globally_. When packages are installed globally, `EACCES` permission errors can occur.
+>
+> Consider setting up npm to operate globally without elevated permissions. See [Resolving Permission Errors](/docs/developing/tips#resolving-permission-errors) for more information.
+
+
+## Create an App
+Next, create an Ionic Vue app that uses the "Tabs" starter template and adds Capacitor for native functionality:
+
+```shell
+$ ionic start photo-gallery tabs --type vue --tag vue-beta --capacitor
+```
+
+This starter project comes complete with three pre-built pages and best practices for Ionic development. With common building blocks already in place, we can add more features easily!
+
+Next, change into the app folder:
+
+```shell
+$ cd photo-gallery
+```
+
+### PWA Elements
+
+Some Capacitor plugins, including the Camera API, provide the web-based functionality and UI via the Ionic [PWA Elements library](https://github.com/ionic-team/pwa-elements).
+
+It's a separate dependency, so install it next:
+
+```shell
+$ npm install @ionic/pwa-elements
+```
+
+After installation, open up the project in your code editor of choice.
+
+Next, import `@ionic/pwa-elements` by editing `src/main.ts`.
+
+```typescript
+// Above the createApp() line
+import { defineCustomElements } from '@ionic/pwa-elements/loader';
+
+// Call the element loader after the platform has been bootstrapped
+defineCustomElements(window);
+```
+
+That’s it! Now for the fun part - let’s see the app in action.
+
+## Run the App
+Run this command in your shell:
+
+```shell
+$ ionic serve
+```
+
+And voilà! Your Ionic app is now running in a web browser. Most of your app can be built and tested right in the browser, greatly increasing development and testing speed.
+
+## Photo Gallery!!!
+
+There are three tabs. Click on the Tab2 tab. It’s a blank canvas, aka the perfect spot to transform into a Photo Gallery. The Ionic CLI features Live Reload, so when you make changes and save them, the app is updated immediately!
+
+![Before and after going through this tutorial](/docs/assets/img/guides/first-app-cap-ng/email-photogallery.gif)
```

Comment:
Update to a gif that show actual vue code, not angular

Reply:
done!